packages/catalog-generator/pom.xml

```diff
@@ -11,10 +11,9 @@
   <description>A Camel Catalog generator for Kaoto.</description>
   <url>https://kaoto.io</url>
   <properties>
-    <compiler-plugin.version>3.13.0</compiler-plugin.version>
```

Comment:
Is this needed? I couldn't see a reference in the `pom.xml`

Reply:
there is already version.maven-compiler-plugin few lines below so i think this one is not used

Reply:
No need if it builds, but maven-compiler-plugin is for sure used in the build. Just default version would be used.